CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [unreleased]
 
+- Fix: Manages responsive of `InfoLines` component.
```

Comment:
Ça se dit pas trop en anglais `Manages responsive of`. On pourrait dire `Handles small version of` plutôt (pareil dans le titre de la PR).